enqueue.go

```diff
@@ -62,12 +72,16 @@ func EnqueueWithOptions(queue, class string, args interface{}, opts EnqueueOptio
 
 	bytes, err := json.Marshal(data)
 	if err != nil {
-		return "", err
+		return "", 0, err
```

Comment:
Not sure what would be better here, but opening up the discussion: would this be better with a sentinel value, like a `-1`? `0` in this context is a valid value so to an unsuspecting caller that doesn't check the `err` may be fooled.

also hi @jrallison! :wave: